python/cucim/src/cucim/skimage/_vendored/_ndimage_filters.py

```diff
@@ -1106,6 +1106,9 @@ def _rank_filter(input, get_rank, size=None, footprint=None, output=None,
             sizes = footprint.shape
             has_weights = False
 
+    if not has_weights:
```

Comment:
Not sure why this shows up here in the diff. The changes in this file are from:
https://github.com/rapidsai/cucim/pull/521

Reply:
resolved now after merging main